spec/facebook_post_generator_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'cute_pets/post_generator'
+require 'minitest/autorun'
+
+describe 'FacebookPostGenerator' do
+  describe '.create_message' do
+    it 'returns a string with the pet name, description and shelter link' do
+      message = PostGenerator.create_message('Moofle', 'neutered female lab', 'http://www.example.org/moofle')
+      message.must_match /Moofle. I am a neutered female lab. http\:\/\/www\.example\.org\/moofle/
+    end
+
+    it 'returns a string with the appropriate article' do
+      message = PostGenerator.create_message('Miffle', 'unaltered female kitten', 'http://www.example.org/miffle')
+      message.must_match /Miffle. I am an unaltered female kitten. http\:\/\/www\.example\.org\/miffle/
+    end
+  end
+
+  describe '.post' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.